File: atcoder.jp/contests/arc026/tasks/arc026_1/main.ts
import { readFileSync } from "fs";
let inputLinesIndex = 0;
const inputLines = readFileSync("/dev/stdin", "utf8").split("\n");
const input = () => inputLines[inputLinesIndex++];

const [N, A, B] = input().split(" ").map(Number);

const b = Math.min(5, N);
const a = N - b;
const ans = A * a + B * b;

console.log(ans);

File: atcoder.jp/contests/arc026/tasks/arc026_1/main.rs
use proconio::input;
use std::cmp;

fn main() {
    input! {
        (n, a, b): (i32, i32, i32),
    }

    let bb = cmp::min(5, n);
    let aa = n - bb;
    let ans = a * aa + b * bb;

    println!("{}", ans);
}
